scripts/build.d/pybind11

```diff
@@ -2,19 +2,45 @@
 
 set -e
 
-pkgname=pybind11
-pkgbranch=${VERSION:-master}
-pkgfull=$pkgname-$pkgbranch
+SRCSYNC=${SRCSYNC:-git}
 
-# unpack (clone)
-syncgit https://github.com/pybind ${pkgname} ${pkgbranch}
+if [ "${SRCSYNC}" == "tarball" ] ; then
+
+  pkgname=pybind11
+  pkgver=${VERSION:-2.7.0}
```

Comment:
Is there any reason why the version is specified to `2.7.0` ?

Reply:
`2.7.0` is the latest release: https://github.com/pybind/pybind11/releases/tag/v2.7.0